stylesheet.css

```diff
@@ -113,7 +117,7 @@ button:active {
   font-weight: 700;
   font-size: 18px;
   line-height: 20px;
-  color: #6070ff;
+  color: #303dab;
 }
```

Comment:
I think you did a great job here. However, there are still some things to improve. There are still issues with the color contrast ratio with some buttons as you can[ see here](https://www.aditus.io/button-contrast-checker/raw-githack-com-2022-11-09-at-13-39-46-034?failing). Kindly fix this. You can use[ this tool ](https://www.aditus.io/button-contrast-checker/)to check it. Feel free to change the color or size text to fulfill the requirement. :+1: :smiley: :smiley: 

![image](https://user-images.githubusercontent.com/98363075/200845318-96963856-fbca-44a3-af6d-f6349c99fd2b.png)

![image](https://user-images.githubusercontent.com/98363075/200845354-e9ba60b9-3210-4454-8fe7-9457570d1036.png)

![image](https://user-images.githubusercontent.com/98363075/200845417-18f1a38a-5ba9-424f-b6b1-510438ba701e.png)